src/agent/block_store_services/block_store_fs.js

```diff
@@ -328,6 +369,9 @@ class BlockStoreFs extends BlockStoreBase {
      */
     async _move_blocks_to_tmfs(block_ids) {
         const moved_block_ids = [];
+        if (this.xattr_fallback) {
```

Comment:
again, not sure we should change how this config behaves, I would leave this out.